prebid-mobile/prebid-mobile.md

```diff
@@ -10,11 +10,19 @@ sidebarType: 2
 
 Prebid Mobile is an open-source library fir iOS and Android that provides an end-to-end header bidding solution for mobile app publishers.
 
+---- ;
```

Comment:
Do we need `---- ; ` here ?
<img width="890" alt="image" src="https://github.com/user-attachments/assets/559ceda1-50e7-4bd4-a2b9-80e913db4cb8" />

Reply:
hm, it should be a long line... I'll take a look